components/CookieConsentConfig.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * @type {UserConfig}
+ */
+const pluginConfig = {
+  current_lang: "en",
+  autoclear_cookies: true, // default: false
+  page_scripts: true, // default: false
+  test: "ciao",
+  force_consent: true, // default: false
+  // mode: 'opt-in'                          // default: 'opt-in'; value: 'opt-in' or 'opt-out'
+  // delay: 0,                               // default: 0
+  // auto_language: null                     // default: null; could also be 'browser' or 'document'
+  // autorun: true,                          // default: true
+  // hide_from_bots: false,                  // default: false
+  // remove_cookie_tables: false             // default: false
+  // cookie_name: 'cc_cookie',               // default: 'cc_cookie'
+  // cookie_expiration: 182,                 // default: 182 (days)
+  // cookie_necessary_only_expiration: 182   // default: disabled
+  // cookie_domain: location.hostname,       // default: current domain
+  // cookie_path: '/',                       // default: root
+  // cookie_same_site: 'Lax',                // default: 'Lax'
+  // use_rfc_cookie: false,                  // default: false
+  // revision: 0,                            // default: 0
+
+  onFirstAction: function (user_preferences, cookie) {
+    // callback triggered only once
+  },
+
+  onAccept: function (cookie) {
+    // ...
```

Comment:
Nothing happens when someone consents to the cookies!

Reply:
The cookies are stored whenever the Accept button is clicked; this function is for loading extra scripts like for the GA4 tag. 

![Screenshot (291)](https://user-images.githubusercontent.com/76250685/211201297-95f51a8d-58d6-4f3b-bbd0-209d92025604.png)

For GA4, the resource you shared, a Google Tag Manager account is to be made, which is linked to the google account so, should I do it with my personal account or is there a G-suit for that?